.github/workflows/reusable-lint.yaml

```diff
@@ -113,10 +112,10 @@ jobs:
           echo $PWD
           ls -l $PWD/*
       - name: Install black
-        if: needs.get-job-flags.outputs.CI_LINT_BLACK == 'true'
+        if: ${{ needs.get-job-flags.outputs.CI_LINT_BLACK == 'true' }}
```

Comment:
Was this working before?

Reply:
I think so?  I am honestly not sure of the specific syntax for github workflow variables, etc.  Seems like there are a lot of ways to specify the same thing...